diagnostic.md

```diff
@@ -55,7 +55,7 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+Absolute path, because it specifies the entire filepath.
```

Comment:
Correct, but more specifically we know this because the first `/`represents the root of the system